data/json/items/armor.json

```diff
@@ -3174,7 +3174,7 @@
         "warmth" : 0,
         "phase" : "solid",
         "environmental_protection" : 0,
-        "encumbrance" : 3,
+        "encumbrance" : 1,
```

Comment:
1 encumbrance at 25 storage is insane. This would be acceptable with old rules (+10 encumbrance for extra item on layer), but now it means you can just stack 2 of those for 50 storage at 3 encumbrance.